packages/ckeditor5-table/theme/tablecellproperties.css

```diff
@@ -26,3 +26,17 @@
 		}
 	}
 }
+
+.ck-content {
+	& .table {
+		&.table-style-block-align-left {
+			margin-left: 0;
+			margin-right: auto;
+		}
+
+		&.table-style-block-align-right {
+			margin-left: auto;
+			margin-right: 0;
+		}
+	}
+}
```

Comment:
These are table-level styles so maybe they should go to `tableproperties.css` instead?